03_Modules/EVDriver/src/module/module.ts

```diff
@@ -167,6 +167,12 @@ export class EVDriverModule extends AbstractModule {
             // TODO determine how/if to set personalMessage
           },
         };
+
+        if (message.payload.idToken.type === IdTokenEnumType.NoAuthorization) {
```

Comment:
IdTokenEnumType is not imported according to the build result. Please add it to the import statements.